src/chrome/content/rules/MicroAd.co.jp.xml

```diff
@@ -0,0 +1,30 @@
+<!--
+	For more MicroAd, Inc related ruleset, see MicroAdInc.com.xml
```

Comment:
Please also update the reference list on MicroAdInc.com.xml
Can you also remove the reference to `MicroAdInc.com.xml` there, because this would be a reference to itself.